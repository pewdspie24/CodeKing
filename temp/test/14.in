23
81
894 435 5372 9346 6206 5172 472 7777 8204 2747 496 5914 3539 9104 1812 429 6720 2643 8370 9547 282 1637 9535 3460 9438 1316 146 823 285 5543 6140 1074 1265 7775 5175 4584 9011 2673 4931 9180 5805 3457 2594 2908 979 9659 8104 8511 9977 7402 4303 2524 6742 9278 7298 5493 7573 208 7975 5433 5455 381 3036 7251 9557 1733 7935 1148 885 3090 7918 9021 6914 1926 6574 5454 2456 6653 5108 4774 9471 
70
4797 7334 6328 7281 9783 3538 5326 1944 8452 931 7835 5457 1023 4537 9382 3643 568 8898 7288 7863 6085 5392 4425 8153 1743 1780 4836 8678 8563 9952 116 5175 3941 503 5705 6844 2038 9604 9993 4192 1701 5535 8178 4630 6367 6091 9147 4371 1127 4023 2553 4464 1352 4546 8911 4769 271 9735 1481 615 147 4971 4133 7731 1708 9596 8076 723 1548 8249 
88
7119 1140 334 5575 1434 4451 5127 6847 7159 6383 5403 6567 3654 6928 9679 9647 6090 5947 3337 7696 1870 137 1436 3166 5063 5487 3485 9619 6424 4852 5782 5979 4577 6209 3554 6136 7651 5036 3743 9612 5997 6866 5303 2151 2935 5660 4470 733 1206 7286 7130 1034 1028 7302 1644 7645 2407 7022 7913 7198 3174 5531 4959 6143 3398 9167 6513 462 8944 9076 5877 5221 4746 9230 2452 9858 5965 7534 710 3113 9504 9303 7411 3437 5444 132 9788 2892 
82
3175 3427 2599 9978 7147 2654 1967 2178 9736 5091 2262 7169 3216 6284 8046 8036 1580 757 6722 4394 314 9917 9735 1191 2506 5060 7093 8058 8361 1027 9822 8198 4125 6068 6131 2441 6139 6395 7855 4401 8736 6153 8610 1560 7839 7472 6113 3680 4359 7257 3170 9228 3272 216 3998 9260 6685 3594 9727 9977 8273 5025 6458 8005 1988 3014 6866 9104 6477 972 9293 3156 2194 5952 5301 9083 7560 2395 2175 4468 5722 526 
100
8255 9755 1006 5974 6407 4560 5258 7563 4037 4127 2300 8112 3636 3043 7885 7633 3547 8681 9691 333 2657 871 7361 8293 9305 1899 8778 4461 3711 353 608 7171 8536 9351 6700 9018 611 5157 1202 1688 7397 8499 5824 1353 6990 551 7377 2390 7246 8259 5785 7086 7888 7542 2618 1887 1416 3554 3775 2164 1158 8018 636 554 6438 4877 2492 3450 5295 2449 964 7905 4324 3873 6843 3088 6865 519 2350 9022 1165 9495 7318 124 8684 3657 2726 6803 1825 1657 8924 2405 7483 6075 9479 3439 5060 9253 7020 2941 
79
2989 8033 9644 6125 1195 8697 5801 1294 1542 7310 4805 4050 6723 8263 2417 5394 9116 9213 6851 8626 5524 4006 4509 5949 4733 513 7709 3031 2051 3417 718 8923 2850 5863 2173 9522 4562 7227 1616 2095 3509 368 4433 7803 9314 3541 8662 7868 9737 2031 637 5620 1911 6253 8710 1930 7955 8782 9508 5690 9323 2363 7808 424 6035 7062 1697 616 2779 8299 9396 2060 4738 2852 7689 6613 2590 4944 5976 
66
8106 7251 5737 893 5917 639 8104 8755 2153 3623 3502 8043 5723 116 6790 4764 8976 654 6782 4282 6468 928 4697 2112 7038 5388 6354 8581 3607 593 2223 2160 9229 495 9303 571 476 8499 3868 7910 3606 1144 2964 7080 5249 8592 7885 1436 7712 7019 5284 1908 5379 4533 738 2316 2005 3105 7345 1835 3573 4996 8867 2164 8480 9483 
63
3487 9843 1522 748 8249 5116 5036 451 9993 5889 4448 9229 4998 7519 2711 5326 3996 2256 9474 6874 4844 6604 5550 4058 3290 3498 5691 2596 6338 353 2395 6547 2937 157 7242 3442 1973 7589 3723 4290 5470 2629 9737 3630 5032 1911 5145 1437 2742 2789 3518 1301 1388 6669 4554 8810 1787 4641 3199 7039 4003 2626 7148 
71
8981 7505 4637 3036 5216 7470 5133 1024 6994 2729 2497 2743 8339 8576 7459 7812 4338 3974 8396 5308 3621 1844 1858 8901 8261 8213 4024 893 4025 6273 4385 6033 4881 4615 3187 6378 5273 3210 939 2115 1323 1142 4636 8207 5851 8735 6805 8703 3679 1528 9370 4672 3599 5321 1107 9334 2532 5371 7800 6452 9468 9349 6978 357 2231 3169 6736 4684 9039 4578 1526 
90
7050 1014 6570 3397 3150 5935 2922 9535 4509 6816 5217 5257 1896 6103 2611 739 8384 210 2276 213 4965 7686 4499 3231 5731 638 4129 7105 5060 6646 5654 3337 6932 9046 6403 4291 5533 7838 7215 7774 915 663 2567 8533 8100 4170 3551 4629 5965 9369 7241 3157 3823 1164 1464 2662 3814 4808 6075 9844 6015 6677 9518 4217 4100 8693 9656 3855 6891 4213 4848 2758 1367 2665 1779 373 2313 1705 4394 8702 6376 3816 2550 8725 7421 8814 8904 9245 9356 4945 
99
4644 6922 3138 1513 7254 9022 938 4365 6958 6504 7113 7346 941 8731 6295 2395 4384 5725 220 6479 226 1633 3567 7599 9313 1541 9290 3673 3828 191 335 8968 8741 7632 3019 9180 3462 9780 5062 5596 8842 6876 4485 6905 4760 6975 4652 9594 6654 6690 6004 1355 6326 7125 9629 8075 1132 6158 3968 1526 2573 2806 1472 7940 3205 856 165 920 248 7404 1863 2314 6335 277 5960 1114 1579 5007 7856 2137 260 203 2679 476 582 9475 6662 6456 1203 4977 9760 8775 4376 2112 508 9138 960 1279 8954 
50
1475 1974 5472 3184 1235 1894 5152 7135 2653 4308 6247 9934 5241 2823 2259 3494 8365 4962 187 6105 7972 2324 316 7425 3912 8945 5377 321 3833 7632 2608 4715 1528 6201 8380 6609 1370 6821 2276 5501 1304 8409 4761 7844 7011 1000 3682 6846 1045 9489 
96
8780 1848 6591 2295 8122 2003 9459 1496 7933 703 8475 3732 1029 540 4457 3834 4376 4161 4938 1875 6631 8261 6513 165 2598 4509 1084 4538 1236 1691 4480 1000 157 4031 9586 8477 8011 4845 2816 9748 4335 4028 1361 3361 518 4531 9549 9710 3758 4784 5369 4179 6380 4263 158 2727 8877 3605 6939 4855 6586 3237 7461 1111 7076 5718 8728 7356 3053 989 3562 5713 733 6157 7058 6165 743 6984 3645 6034 3154 8742 4050 6852 9695 7623 8666 8686 3104 2828 5252 2089 3755 6702 804 8013 
56
6830 8000 9015 3680 5083 377 3978 2412 4924 5417 521 135 6546 6437 4895 1201 2762 7442 536 6508 9109 8234 3924 6906 2123 2526 9668 101 4739 8844 8503 4989 5583 8045 918 7690 380 7383 9683 1959 9790 6775 6916 3144 9630 8967 4221 2498 8692 3950 9003 2691 4240 4312 6404 8636 
77
9139 2140 1176 7504 3335 4548 2956 1877 1632 4657 3753 7890 1920 1455 7375 9533 8288 3245 4672 6525 558 2927 5475 2138 1965 1980 4655 8363 3660 3600 8934 3625 8633 2827 5499 3641 2012 2434 7418 4376 4834 518 3638 6202 2353 6118 4112 480 8738 2580 2610 575 8401 311 2704 2518 6785 2005 5038 4336 4947 2865 8089 3197 2043 9819 7097 1469 4496 9480 5241 8830 8193 128 9941 5607 5631 
80
6803 9919 5030 3019 5761 7690 236 6928 989 2161 9169 1315 8293 735 8835 5348 4693 485 8443 3909 7242 8574 1758 2983 3508 3754 2928 4857 9255 2436 3930 5122 148 6265 1868 8349 6206 8727 3642 3767 4499 1869 1649 8271 8620 3863 7318 9406 7220 807 7636 6792 2903 9027 7581 2030 1531 4302 7904 7170 5980 6909 5901 9736 7632 5231 3953 3341 8661 9951 8098 8561 5914 7710 9043 8650 3613 3366 9953 224 
79
3158 6060 6792 9291 8624 8940 5289 4274 3269 7026 7312 8734 9327 756 3822 4952 7809 9676 1822 5270 7636 3788 7810 173 2043 8776 2855 7676 2157 1085 1181 6581 6200 9206 1751 6677 2221 4727 9425 3968 8781 5581 239 9937 9808 578 6937 528 8523 2895 2208 1323 6062 7271 2864 2032 8284 1875 5334 6429 7001 1248 9927 7928 7100 5820 5708 6909 2463 6520 4668 6183 2745 4173 6697 185 4516 9021 8145 
74
2570 4178 3453 1239 5093 9160 3505 1503 4527 9593 8990 8488 2808 4359 7489 9615 5825 8851 6459 279 7025 6931 8812 2563 8670 1782 1406 5056 2045 9390 2744 5667 8135 7920 3372 4704 2048 5681 1924 6080 7144 8218 7675 6615 7561 3404 2498 4806 7701 3627 2720 4535 5173 8624 1043 5085 8771 4583 6706 5131 3466 2493 5848 4990 2590 2550 4739 1637 3214 6835 8753 4387 2323 6631 
76
1827 2777 4296 6807 2896 3659 6277 5727 8290 2247 7572 6170 4781 8421 2708 4112 3573 3503 7670 4987 433 3962 1076 1530 3114 6241 9500 2603 8763 8952 2194 7841 9036 7139 5086 1087 1342 3596 405 6851 9899 7063 188 348 5188 243 5009 9962 8433 7319 4229 2473 7295 201 3285 3372 8625 5793 1404 8666 5164 4597 6608 593 2582 7632 7721 7349 2690 3935 891 8412 1804 4954 2829 7830 
83
196 1230 6169 884 553 2296 3056 8327 9235 1092 7749 7043 2307 7369 256 2212 5222 2060 9785 6060 342 5946 3780 8432 898 6986 3197 3236 4036 7169 8665 1260 3871 6816 435 3814 3277 141 4314 7377 982 3716 3867 9052 6905 3493 2552 3273 9538 8868 876 680 4385 6536 6769 8269 9482 1091 8813 2582 2793 5554 9776 7228 9206 4697 405 6529 9711 7461 3423 793 3403 538 4682 227 3329 6154 9295 7898 1785 9687 3254 
70
6688 3268 7550 940 6904 9775 3742 2881 256 8828 8562 9064 2503 9529 5578 7416 6609 9073 1280 3582 1621 3638 3599 4985 9944 3247 9861 4209 4584 1741 7119 2459 1284 2221 3183 625 5593 1151 9237 831 9451 9699 8661 9754 7711 7152 962 3867 2219 6506 2201 5391 1376 3875 562 1677 2528 5795 9924 2813 3265 5348 6163 2146 1386 9855 1391 7064 6933 842 
80
8376 9873 6454 3004 358 3603 7172 2792 5272 4128 2433 7266 6524 3769 2762 5208 6387 496 7209 6052 7699 3418 4609 4884 3905 1759 5723 7524 8332 6437 5264 3851 8072 5947 1878 7748 5672 2893 5976 5135 5044 5914 8644 3948 7467 8112 6040 8867 4716 2761 5043 7460 1204 2583 8078 4102 872 4431 2666 3193 1762 2095 1905 3795 1911 7978 3455 9589 2365 9829 3775 9356 6593 2831 8438 9724 9813 1620 1720 305 
55
5241 3741 4746 1428 9129 6332 849 986 3614 7732 8551 7028 2961 7130 5031 4070 7936 503 9517 7279 2920 6484 757 6458 7821 352 7635 4567 8275 7856 3650 1313 572 3768 1827 1793 4744 2060 8312 6291 9902 6645 895 9223 8567 3224 5405 9423 5429 6785 6692 1391 7406 9983 111 
