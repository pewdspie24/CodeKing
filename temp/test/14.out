Bi
Lema
Bi
Bi
Lema
Bi
Lema
Bi
Lema
Bi
Lema
Bi
Bi
Bi
Bi
Lema
Bi
Lema
Bi
Bi
Bi
Lema
Bi
